app/controllers/application_controller/automate.rb

```diff
@@ -30,7 +30,7 @@ def resolve_button_throw
       else
         page.replace("left_cell_bottom", :partial => "resolve_form_buttons")
         page.replace("flash_msg_div", :partial => "layouts/flash_msg")
-        page.replace_html("main_div", :partial => "results_tabs")
+        page.replace_html("simulate_result", :partial => "results_tabs")
```

Comment:
Seems like we might need `params[:controller] == "miq_ae_tools" ? "simulate_result" : "main_div"`..